src/BenchmarksServer/Startup.cs

```diff
@@ -2566,6 +2566,11 @@ private static async Task<string> CloneRestoreAndBuild(string path, ServerJob jo
                 }
 		
                 await UseMonoRuntimeAsync(runtimeVersion, outputFolder, job.UseMonoRuntime);
+
+		if (job.UseMonoRuntime.Equals("llvm-aot"))
```

Comment:
You can call this method from `UseMonoRuntimeAsync` directly

Reply:
I think this it makes more sense to call it here, because AOT4Mono is a post-process for the app publish folder.

Reply:
So is the rest of `UseMonoRuntimeAsync` which already does things specific to `"llvm-aot"`. It's on the line right above, it's still post publication.

Reply:
Yes, I understand that it is post publication. UseMonoRuntimeAsync is a function to switch the runtime to using runtime in the publish folder. AOT4Mono is a function to pre-compile all assemblies in the publish folder. I don't see the first one could contain the second one. As they are parallel to each other. If you rename `UseMonoRuntimeAsync` to `PostProcess4Mono`. It could make sense to put both `UseMonoRuntimeAsync` and `AOT4Mono` inside. In that case, I will probably still create a separate function for `UseMonoRuntimeAsync` to make things more clear, according to single-responsibility principle.